ambassador/entrypoint.sh

```diff
@@ -392,6 +392,7 @@ if [[ -z "${AMBASSADOR_NO_KUBEWATCH}" ]]; then
            ${KUBEWATCH_SYNC_KINDS} \
            ${AMBASSADOR_FIELD_SELECTOR_ARG} \
            ${AMBASSADOR_LABEL_SELECTOR_ARG} \
+           --interval ${AMBASSADOR_WATCH_INTERVAL_MS:-"250"}ms \
```

Comment:
For watt it's worth, you don't need to quote the 250 here.

Reply:
Is that a "watt" pun? If so, excellent.

Reply:
Rebased, and removed quotes. Let me know if you have any other comments.